### PR TITLE
document about Helm dashboard

### PR DESCRIPTION
**Pull request by pushker001**
## Changes Proposed
help old and new user to know the capability of helm dashboard

## CheckList

- [X] The title of my pull request is a short description of the changes
- [X] This PR relates to some issue:

### CODE REVIEW

**Comment by hdm23061993** [comment]
tHIS IS RELATED TO ISSUE #255

**Comment by pushker001** [comment]
Screenshot (342)


> This is related to #255

yes

**Comment by pushker001** [comment]
> This is related to #255

yes

File: FEATURES.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * POST /api/releases: Installs a new release using a specified chart and configuration.
+ * DELETE /api/releases/{release_name}: Deletes an existing release.
+
+ 3.  The endpoints in the REST API may accept various parameters, such as release names, chart names, and configuration values. These parameters are typically specified as query parameters or in the request body, depending on the endpoint.
+
+4. Authentication: The REST API may require authentication to access certain endpoints, depending on the configuration of the Helm Dashboard. The API supports several authentication methods, including basic authentication and bearer tokens.
+
+5. Response format: The REST API typically returns responses in JSON format, which can be easily parsed by client applications.
+
+
```
Comment (hdm23061993) [inline FEATURES.md:48]: Please remove these extra lines

**Pull request closed without merging**